### PR TITLE
[Fixes #804] Re-enable tests disabled by cookie path changes

### LINKED ISSUE #804: Re-enable tests disabled by cookie path changes
External issue: https://github.com/dotnet/corefx/issues/18013 & https://github.com/dotnet/corefx/issues/21250

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

File: test/MusicStore.E2ETests/Implementation/OpenIdConnectLoginScenarios.cs [hunk 1]
```diff
@@ -98,12 +107,21 @@ public partial class Validator
...
             handler.CookieContainer.Add(new Uri(_deploymentResult.ApplicationBaseUri), _httpClientHandler.CookieContainer.GetCookies(new Uri(_deploymentResult.ApplicationBaseUri)));
             _httpClient = new HttpClient(handler) { BaseAddress = new Uri(_deploymentResult.ApplicationBaseUri) };
 
             response = await DoPostAsync("Account/LogOff", content);
+            Assert.True(response.Headers.TryGetValues("Set-Cookie", out var logoutSetCookieHeaders));
+            var responseCookies = SetCookieHeaderValue.ParseList(logoutSetCookieHeaders.ToList());
+
+            foreach (var logoutSetcookie in logoutSetCookieHeaders)
+            {
+                _httpClientHandler.CookieContainer.SetCookies(new Uri(_deploymentResult.ApplicationBaseUri), logoutSetcookie.Replace("; samesite=lax", ""));
```
Comment (pranavkm) [inline test/MusicStore.E2ETests/Implementation/OpenIdConnectLoginScenarios.cs]: cc @Tratcher

Comment (Tratcher) [inline test/MusicStore.E2ETests/Implementation/OpenIdConnectLoginScenarios.cs]: So how is this better than just ditching CookieContainer?

Comment (javiercn) [inline test/MusicStore.E2ETests/Implementation/OpenIdConnectLoginScenarios.cs]: I'm just trying to fix the test. I leave any bigger change to the discretion of @kichalla. Flowing the cookies manually is not a trivial process and is error prone.

File: test/MusicStore.E2ETests/SmokeTestRunner.cs [hunk 2]
```diff
@@ -148,7 +148,20 @@ public SmokeTestRunner(ITestOutputHelper output) : base(output)
...
+
+            logger.LogInformation("Verifying Twitter login scenarios");
+            await validator.LoginWithTwitter();
+
+            logger.LogInformation("Verifying Microsoft login scenarios");
+            await validator.LoginWithMicrosoftAccount();
+
             logger.LogInformation("Variation completed successfully.");
         }
+
```
Comment (pranavkm) [inline test/MusicStore.E2ETests/SmokeTestRunner.cs]: Remove.

**Pull request merged**